gtsam/hybrid/IncrementalHybrid.h

```diff
@@ -50,6 +50,18 @@ class IncrementalHybrid {
   void update(GaussianHybridFactorGraph graph, const Ordering& ordering,
               boost::optional<size_t> maxNrLeaves = boost::none);
 
+  /**
+   * @brief Method to prune the decision tree of discrete variables.
+   *
+   * @param maxNrLeaves The maximum number of leaves to keep.
+   * @return DecisionTreeFactor::shared_ptr 
+   */
+  DecisionTreeFactor::shared_ptr prune(size_t maxNrLeaves);
```

Comment:
not const. I assume this will be resolved in #44

Reply:
Resolved